packaging/opensuse/snapd.spec

```diff
@@ -51,29 +56,18 @@
 # Additional entry of $GOPATH during the build process.
 # This is designed to be a sub-directory of {_builddir}/{name}-{version}
 # because that directory is automatically cleaned-up by the build process.
-%global         indigo_gopath   %{_builddir}/%{name}-%{version}/gopath
+%global indigo_gopath   %{_builddir}/%{name}-%{version}/gopath
```

Comment:
Why are we calling this indigo? This isn't really the indigo thing, is it?

Reply:
In a way it is not, indigo is after figuring out how to pass compiler flags that are appropriate. In another way it is. The two variables, `indigo_gopath` and `indigo_srcdir` define the GOPATH for the build process and root of the source tree of the current project. Indigo doesn't use them yet but I am thinking of a way to do so.